src/app/api/services/dot-workflow/dot-workflow.service.ts

```diff
@@ -44,4 +44,20 @@ export class DotWorkflowService {
             })
             .pluck('entity');
     }
+
+    /**
+     * Updates the workflow actions for a page asset
+     *
+     * @param {string} inode
+     * @returns {Observable<any[]>}
+     * @memberof DotWorkflowService
+     */
+    updateWorkflowActions(inode: string, actionId: string): Observable<any[]> {
```

Comment:
This is not "update" the action is fire. So maybe we can call this method: ```fireWorkflowAction```

Reply:
why any[]?, I think this end point return a contentlet, We have to create a contentlet interface

Reply:
What happen with this method's test?